cla-backend/cla/config.py

```diff
@@ -77,6 +77,9 @@
 EMAIL_SERVICE = 'SNS'  #: Email service to use for notification emails.
 EMAIL_ON_SIGNATURE_APPROVED = True  #: Whether to email the user when signature has been approved.
 
+# Platform Maintainers
+PLATFORM_MAINTAINERS = ["ddeal@linuxfoundation.org", "makkalot@gmail.com"]
+
 # SMTP Configuration.
 #: Sender email address for SMTP service (from address).
 SMTP_SENDER_EMAIL_ADDRESS = 'test@cla.system'
```

Comment:
I know this wasn't part of your issue/PR, but can we also load this from AWS SSM? We may need to create a new parameter.

Reply:
sure 👍